masonry/src/widget/widget.rs

```diff
@@ -199,6 +199,7 @@ pub trait Widget: AsAny {
     /// As methods recurse through the widget tree, trace spans are added for each child
     /// widget visited, and popped when control flow goes back to the parent. This method
     /// returns a static span (that you can use to filter traces and logs).
+    // TODO: Make include the widget's id?
```

Comment:
Agree, could probably be done in this PR.

Reply:
That's an extremely significant change, because the widget's id isn't already in scope. I don't want to do this in this PR.

Reply:
Oh, right, we're in the widget trait, I guess I missed that. Probably needs the id as function parameter or something like that. Sure let's keep that for a follow-up PR